pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -222,13 +222,11 @@ def set_epoch(self, x_axis_var):
                         delta_time = (delta_plus_var
                                       - delta_minus_var) / 2.0
```

Comment:
This variable is no longer used anywhere.

Reply:
That's true. After the last discussion I heard about this, It was my understanding that I was going to attempt to convert this to a datetime to be added to the result of cdflib.cdfepoch.to_datetime(xdata).  Not sure if that's correct or not. The hold up is that I haven't figured out if this variable always has the same initial units or not so I haven't pushed through a fix. I had previously removed this variable and all of the code that goes into determining delta_plus_var and delta_minus_var, but that removed a lot of code

Reply:
I am trying to do a CINDI analysis at the moment and CDFLIB appears to be broken for our implementation.  May or may not be related to this, but I am trying to fix things.

Reply:
What are the units of this variable?

Reply:
Nevermind, figured it out.